apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/model/SimpleXAPI.kt

```diff
@@ -6442,6 +6452,8 @@ data class AppSettings(
         iosCallKitEnabled = true,
         iosCallKitCallsInRecents = false,
         uiProfileImageCornerRadius = 22.5f,
+        uiChatItemRoundness = 0.75f,
```

Comment:
don't forget to update core (so AppSettings can be saved/restored) and provide the same default on every platform

Reply:
will be available as soon as https://github.com/simplex-chat/simplex-chat/pull/4977 is merged